### PR TITLE
Add final touch

### PR DESCRIPTION
**Pull request by farouqdaif1**
* Used the information detailed in Zeplin to style the website.
* Updated README file with screenshot and live demo link

### CODE REVIEW

File: src/components/bookComponents/renderbook.js [hunk 1]
```diff
@@ -8,10 +8,22 @@ const Book = (props) => {
...
+          {` ${book.title} `}
+        </span>
+      </div>
+      <div className="btn-sec">
+        <button type="button" className="btn-c"> Comments </button>
+        <div className="line-2" />
+        <button type="submit" className="remove-btn btn-r" onClick={() => dispatch(removeBook(book))}>REMOVE</button>
+        <div className="line-2" />
+        <button type="button" className="btn-e"> Edit </button>
+      </div>
```
Comment (omar-labana) [inline src/components/bookComponents/renderbook.js]: - [ ] Currently the progress bar and chapter info are missing from each book. Please add them to match the given design.

File: src/components/bookComponents/addBook.js [hunk 2]
```diff
@@ -19,10 +19,10 @@ const Form = () => {
     dispatch(addBook(newBook));
   };
   return (
-    <form id="add-book-form" onSubmit={(e) => submitForm(e)}>
+    <form id="add-book-form" className="add-book-form" onSubmit={(e) => submitForm(e)}>
```
Comment (omar-labana) [inline src/components/bookComponents/addBook.js]: - [ ] Please add the horizontal rule between the add book form and the list of books. Also, please add the missing header `ADD NEW BOOK` for this section.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @farouqdaif1, 

Excellent work so far! 👌
You have correctly implemented all of the requirements and have demonstrated your understanding of React. 👏. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/bookComponents/addBook.js (hunk 2 above)
Comment (farouqdaif1) [reply src/components/bookComponents/addBook.js]: thanks

File: src/components/bookComponents/renderbook.js (hunk 1 above)
Comment (farouqdaif1) [reply src/components/bookComponents/renderbook.js]: thanks

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @farouqdaif1,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**